### PR TITLE
Porting to MVP design pattern.

### LINKED ISSUE #110: Refactor app's code base to follow MVP framework provided by Ribot
## Description
As a developer,
I need to refactor code,
so that code base follows MVP framework provided by Ribot.

## Acceptance Criteria
The code should follow the MVP framework

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentor.

## Estimation
4-5 days

### PR DESCRIPTION
**Pull request by haroonshah2764**
### Description
Porting to MVP design pattern.
Fixes #110 

### Type of Change:
**Delete irrelevant options.**

- Code

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
- This change requires a documentation update (software upgrade on readme file)
- New feature (non-breaking change which adds functionality pre-approved by mentors)

### Checklist:
**Delete irrelevant options.**

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials

### CODE REVIEW

**Comment by haroonshah2764** [comment]
@ExactlyMay @roopalJazz @manjumegh 
work is complete please review

**Comment by haroonshah2764** [comment]
@roopalJazz Butterknife doesn't allow using access modifiers with views that it injects.
these codacy issues cannot be fixed.

File: app/src/main/java/vola/systers/com/android/activities/EventDetailViewActivity.java [hunk 1]
```diff
@@ -1,230 +1,230 @@
...
-        switch (id) {
-            case R.id.btn_register:
-                registerEvent();
-                break;
-            case R.id.fab:
-                bookMarkEvent();
-                break;
-        }
-    }
+//import android.support.v7.widget.Toolbar;
```
Comment (manjumegh) [inline app/src/main/java/vola/systers/com/android/activities/EventDetailViewActivity.java]: Its not acceptable to commit dead code (commented code, almost in entire file). Please remove this and completely and send only working code.

Comment (haroonshah2764) [reply app/src/main/java/vola/systers/com/android/activities/EventDetailViewActivity.java]: I totally agree with you thats what i suggested earlier. but when i asked this publicly on slack i was asked not to remove the code but comment it.

**Pull request closed without merging**